### ISSUE TITLE
What's new in V6 - Element IDs are not unique

### ISSUE THREAD

**Issue by yesenia-gtz** [Office-of-Digital-Services/California-State-Web-Template-Website#956]
Element IDs are not unique

- [x] Two or more elements on this page have the same id value. These should be unique. Make one of the ID elements unique. 

![image](https://user-images.githubusercontent.com/98193284/207424574-e18575a8-3944-4dae-b6dd-7b016e27ce27.png)

**Comment by extinah** [comment]
@pglomboy ready for your review :)

**Comment by pglomboy** [comment]
can I have the url on this please :)

**Comment by yesenia-gtz** [comment]
@pglomboy https://as-cdt-pub-migr-betaws-ww-p-001-stage.azurewebsites.net/whats-new.html

**Comment by pglomboy** [comment]
Please fix this issue: 
![image](https://user-images.githubusercontent.com/120594656/208203076-86d78b39-cbe7-417a-8dcc-f811d30779f7.png)
Components txt link go to different direction, make one of the text link unique

**Comment by yesenia-gtz** [comment]
@jon-grant-state-ca-gov  please see review comments above.

**Comment by extinah** [comment]
@jon-grant-state-ca-gov I'm going to talk to Pia about this one. It's the same link text.

**Comment by extinah** [comment]
@jon-grant-state-ca-gov: Please change "Components" heading to "Component updates"
cc: @pglomboy 

I'll make edits in google and figma. Thanks!

**Comment by kkoryaka** [comment]
@extinah I am not recommending changing any real live text. I would add screen reader's text to solve this (or aria-labels),.

**Comment by extinah** [comment]
@kkoryaka  I worked through the accessibility part of it with Pia and Koji. We talked about aria labels and what would be best for people. We decided ultimately to update the link text/heading is ultimately the best thing to do. When possible, it's best to avoid using aria-labels.

**Comment by kkoryaka** [comment]
@extinah which link you wanted to change: site navigation or page navigation?

**Comment by kkoryaka** [comment]
@extinah need h2 solution for patterns right now. 
![Screen Shot 2022-12-19 at 3 22 59 PM](https://user-images.githubusercontent.com/31669748/208551032-600ce4f1-8b56-4f61-978a-9d539ee7b200.png)

**Comment by extinah** [comment]
Thanks Kon. Let's do "New patterns section"

**Comment by extinah** [comment]
Reviewed and approved. Thanks. Ready to close.
